eng/psakefile.ps1

```diff
@@ -30,15 +29,19 @@ task run-manylinux-container-image -preaction { Write-CacheStats } -postaction {
     # docker will create it and it will be owned by root and
     # the caching/install breaks with permission errors.
     # New-Item is idempotent so we don't need to check for existence
-    $cacheMount, $cacheEnv = Get-CCacheParams
+    $llvmDir = Resolve-InstallationDirectory
+    if (!(Test-Path $llvmDir)) {
+        New-Item -ItemType Directory -Force $llvmDir | Out-Null
+    }
```

Comment:
This is used in 3 places, would be nice to have a function that both creates and returns an installation directory.